code/modules/mob/living/silicon/robot/drone/maint_drone.dm

```diff
@@ -179,10 +182,7 @@
 
 			if(emagged)
 				return
-			if(allowed(I))
-				shut_down()
-			else
-				to_chat(user, "<span class='warning'>Access denied.</span>")
+			shut_down()
```

Comment:
We should probably check if they're already pathfinding before asking the user, so that people cant generate 1000 pathfinding requests